developer-hooks.sh

```diff
@@ -2,21 +2,28 @@
 
 ## Pre-commit hook validation script for the developers.
 
+
 ## Variables
-HOOK_FILE="https://raw.githubusercontent.com/Flux7Labs/central-repo-git-hooks/blob/master/.pre-commit-config.yaml"
+HOOK_FILE_URL="https://raw.githubusercontent.com/Flux7Labs/central-repo-git-hooks/master/.pre-commit-config.yaml"
 ARGS="--all-files"
 PKGS="pre-commit cfn-lint flake8 checkov"
+BASEDIR="/tmp/hooks"
+HOOK_FILE="${BASEDIR}/hookfile"
+
+# create directory to store hook file temporarily
+mkdir -p ${BASEDIR}
+
+# cURL to fetch the hook file
+curl ${HOOK_FILE_URL} > ${HOOK_FILE}
 
-## Validate the HOOK_FILE ResponseCode.
-RESP_CODE=$(curl -o /dev/null -s -w "%{http_code}\n" ${HOOK_FILE})
 
-if ${RESP_CODE} == 200; then
-    ## Pre-commit install and validation
-    pip install ${PKGS}
+if [[ -f ${HOOK_FILE} ]]; then
```

Comment:
```suggestion
if [[ -s ${HOOK_FILE} ]]; then
```
Lets also check the file is not empty.

Reply:
Changes updated as requested